lib/fs-watch/src/lib.rs

```diff
@@ -1,6 +1,5 @@
 #[macro_use]
 extern crate log;
-#[macro_use]
```

Comment:
I suspect this macro isn't used on Mac, but it is on Linux.

Reply:
Yeah, that's correct.